configs/config.protectli_vp66xx

```diff
@@ -23,8 +23,8 @@ CONFIG_PCIEXP_LANE_ERR_STAT_CLEAR=y
 CONFIG_DRIVERS_EFI_VARIABLE_STORE=y
 CONFIG_DRIVERS_EFI_FW_INFO=y
 CONFIG_DRIVERS_EFI_MAIN_FW_GUID="dbe538fe-d0fa-4a33-aa36-1d32945e4154"
-CONFIG_DRIVERS_EFI_MAIN_FW_VERSION=0x00090301
-CONFIG_DRIVERS_EFI_MAIN_FW_LSV=0x00090300
+CONFIG_DRIVERS_EFI_MAIN_FW_VERSION=0x00090302
+CONFIG_DRIVERS_EFI_MAIN_FW_LSV=0x00090302
```

Comment:
That was the last version we can downgrade to via capsule? Is it useful for testing to leave `RC0` here?